app.rb

```diff
@@ -49,10 +49,22 @@ def update_won_games_count
       session[:games_won] += 1
     end
 
+    def keep_same_win_counts(game)
```

Comment:
I assume this method is there to fix the issue that it increments the win count every time you go to the /end-of-game route... but it seems kind of like a hack! It's not very intuitive to understand when you see that you're subtracting 1 from the number of games won. Is there a different way to solve this problem?

Reply:
I couldn't find another solution 😭 
I hope to come up with something else while styling the app...